cppcache/src/statistics/StatisticsManager.hpp

```diff
@@ -71,6 +71,8 @@ class StatisticsManager {
                     CacheImpl* cache, const char* durableClientId,
                     const std::chrono::seconds durableTimeout,
                     int64_t statFileLimit = 0, int64_t statDiskSpaceLimit = 0);
+  StatisticsManager(const StatisticsManager& statisticsManager) = default;
```

Comment:
Why are we adding copy constructors? There should only be one `StatisticsManager` per `Cache`. We don't ever want it to copy.